### PR TITLE
Change structure of test data

### PR DESCRIPTION
**Pull request by mloviska**
- Verification run: [bci_repo](http://kepler.suse.cz/tests/16669#step/bci_repo/273)

### CODE REVIEW

File: tests/containers/bci_repo.pm [hunk 1]
```diff
@@ -62,12 +77,13 @@ sub run {
...
         container_exec($container, 'zypper lr -d');
-        record_info("Patterns", join(', ', @{$patterns->[$i]}));
+        record_info("Patterns", join(', ', @{$tdata->[$i]->{patterns}}));
         container_exec($container, 'zypper search -t pattern', timeout => 180);
-        container_exec($container, "zypper -n in -t pattern @{$patterns->[$i]}", timeout => 600);
+        container_exec($container, "zypper -n in -t pattern @{$tdata->[$i]->{patterns}}", timeout => 600);
         container_exec($container, "zypper -q -s 11 pa --orphaned | grep -v sles-release| tee -a repo.test", timeout => 600);
         container_exec($container, 'sed -i "s/ \+/ /g" repo.{org,test}', timeout => 600);
         container_exec($container, "diff repo.org repo.test", timeout => 600);
+        container_exec($container, "rpm -q @{$tdata->[$i]->{packages}}", timeout => 600);
```
Comment (jlausuch) [inline tests/containers/bci_repo.pm:86]: is it possible to print the output of this with `record_info`?

**Comment by jlausuch** [comment]
This is awesome!

File: tests/containers/bci_repo.pm (hunk 1 above)
Comment (mloviska) [reply tests/containers/bci_repo.pm:86]: Yes, but I would need to refactor `container_exec`. I do not really see the added value atm, but ofc it is possible.

Comment (jlausuch) [reply tests/containers/bci_repo.pm:86]: ah ok, right, let's ditch it, it was just to highlight the output.

**Review by jlausuch — APPROVED** [review]

**Pull request merged**